firmware/application/ui/ui_menu.cpp

```diff
@@ -251,8 +251,12 @@ bool MenuView::on_key(const KeyEvent key) {
         case KeyEvent::Down:
             return set_highlighted(highlighted_item + 1);
 
-        case KeyEvent::Select:
         case KeyEvent::Right:
+            if (on_left) {
```

Comment:
It's a nice change. 
sorry for my stupid question, but I don't understand why on_left in right case? maybe a typo? since you declared on_right but never use in code.

Reply:
Damn I didn't changed the var after copy paste, nice catch !
Let me fix this quickly

Reply:
Fixed